Runtime/Exporters/Exporter.cs

```diff
@@ -61,17 +61,17 @@ public static void Export(TreasuredMap map)
 
         public static void ForceExport(TreasuredMap map)
         {
-            if (string.IsNullOrWhiteSpace(map.exportSettings.folderName))
+            if (string.IsNullOrWhiteSpace(map.projectFolder))
             {
                 throw new ArgumentException($"Export Settings > Folder Name is empty.");
             }
             var exporters = ReflectionUtils.GetSerializedFieldValuesOfType<Exporter>(map);
             DataValidator.ValidateMap(map);
-            if (Directory.Exists(map.exportSettings.OutputDirectory))
+            var exportPath = Path.Combine(map.projectFolder, ".treasured", map.projectFolder);
```

Comment:
Should be using Map.exportSettings.OutputDirectory to make it backward compatible